sample/EOLHostMigration/CheckEOLAppsPerAzSub.ps1

```diff
@@ -0,0 +1,21 @@
+# This script lists outs Azure Function apps that are running on the v3 version of the host so they can be upgraded  
+# Review https://learn.microsoft.com/en-us/azure/azure-functions/migrate-version-3-version-4 for details on how to upgrade to ~4 of the runtime 
+ 
+$Subscription = '<YOUR SUBSCRIPTION ID>' 
+ 
+Set-AzContext -Subscription $Subscription | Out-Null 
+ 
+$FunctionApps = Get-AzFunctionApp  
+ 
+$AppInfo = @{} 
+foreach ($App in $FunctionApps) 
+{ 
+    $AppSettings = Get-AzFunctionAppSetting -ResourceGroupName $App.ResourceGroupName -Name $App.Name 
```

Comment:
This is not needed. You already have access to the app settings in the `$app` object: `$App.ApplicationSettings["FUNCTIONS_EXTENSION_VERSION"]`

Reply:
it didn't print anything for some reason... I tested both. Anyways I am moving this request to a different repo and I am adding a CLI example, I will group you and Fabio for a review in PM. thanks